app/views/shared/_modal_form.html.erb

```diff
@@ -9,12 +9,13 @@
         </div>
         <%= simple_form_for [@product, @review] do |f| %>
           <div class="modal-body">
-            <%= f.input :rating, input_html: { min: '1', max: '5' }, required: true %>
+            <%= f.input :rating, input_html: { min: '1', max: '5' }, required: true, label: 'Rating (1-5)' %>
             <%= f.input :comment, as: :text, required: true %>
+
           </div>
           <div class="modal-footer">
-            <button type="button" class="btn btn-secondary" data-dismiss="modal">Close</button>
-            <%= f.submit "Submit review", class: "btn btn-primary" %>
+            <button type="button" class="btn btn-dark" data-dismiss="modal">Close</button>
+            <%= f.submit "Submit review", class: "btn btn-dark" %>
```

Comment:
Secondary color as before sounds better. The button should be able to tell its purpose by just color too.